CHANGES.txt

```diff
@@ -38,6 +38,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       User-facing behavior does not change with this fix (GH Issue #3726).
     - Fix occasional test failures caused by not being able to find a file or directory fixture
       when running multiple tests with multiple jobs.
+    - Added support for a new command-line parameter "--hash-format" to override the default
+      hash format that SCons uses. It can also be set via SetOption('hash_format'). Supported
+      values include md5, sha1, and sha256, but you can also use any other algorithm that is
+      offered by your Python interpreter's hashlib package.
```

Comment:
Add a blurb here about the file naming based on the hash selected

Reply:
Also, please add a node that the default Decider is now called 'content' and not 'MD5' (though md5 will continue to work).